tslint.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "tslint:recommended",
+  "extends": ["tslint-eslint-rules", "tslint:recommended"],
   "rules": {
     "array-type": false,
     "arrow-parens": false,
```

Comment:
Just wondering - did you create all these rules by yourself? Or is there a common angular code style guide, which you reused?

Reply:
I write only part rules. Main part generated with angular cli.